assets/js/translator.js

```diff
@@ -133,8 +133,6 @@ if(modeEnabled('translation')) {
             synchronizeTextareaHeights();
         });
 
-        $(window).resize(synchronizeTextareaHeights);
```

Comment:
Hmm yeah. But I believe removing it should not cause an issue right? As in, it is just synchronizing heights! We could add it very easily in the event we need it. Inputs?